tests/syntax.sh

```diff
@@ -15,6 +15,14 @@ log_sfx()
 	printf "%.128s" "$*" | tr -c '0-9A-Za-z-=,' '_'
 }
 
+check_zero()
+{
+	$STRACE "$@" 2> "$LOG.$sfx" > /dev/null || {
```

Comment:
Looks like `$sfx` is undefined here, see how it's defined e.g. in `check_h`.

Reply:
Updated:
```
check_zero()
{
	local sfx
	sfx="$(log_sfx "$*")"
```